doc/tasks.rst

```diff
@@ -198,8 +198,8 @@ You can also yield a list of tasks.
         def run(self):
             other_target = yield OtherTask()
 
-	    # dynamic dependencies resolve into targets
-	    f = other_target.open('r')
+        # dynamic dependencies resolve into targets
+        f = other_target.open('r')
```

Comment:
for the documentation code example here, should these two lines be indented _inside_ of the `run()`? Looks like you just bumped them outside.

Reply:
True! Corrected in 8c3e3a54.